### PR TITLE
CRM-18405: When creating a contribution in the back-end, total amount  is incorrect

### PR DESCRIPTION
**Pull request by monishdeb**
- [CRM-18405: When creating a contribution in the back-end, total amount is incorrect](https://issues.civicrm.org/jira/browse/CRM-18405)

### CODE REVIEW

**Comment by monishdeb** [comment]
@eileenmcnaughton the current patch works for me

File: templates/CRM/Contribute/Form/Contribution.tpl [hunk 1]
```diff
@@ -608,11 +608,14 @@ function showStartDate( ) {
 }
 
 {/literal}{/if}{literal}
+var thousandMarker = "{/literal}{$config->monetaryThousandSeparator}{literal}";
+var seperator = "{/literal}{$config->monetaryDecimalPoint}{literal}";
```
Comment (eileenmcnaughton) [inline templates/CRM/Contribute/Form/Contribution.tpl]: Argh repeat after me sepArator

someone mispelt that in the code base ages ago & now everyone thinks it's the right spelling!

**Comment by monishdeb** [comment]
Oops blind cut paste :D  Updated the PR

**Comment by eileenmcnaughton** [comment]
The patch reads well to me - I haven't tested it though & I guess the original submitter will probably do that & confirm

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by eileenmcnaughton** [comment]
I tested this as follows:

1) set site default thousand separator to . & decimal separator to ,
2) Entered a back office contribution of 5.000,60
3) enter a fee amount of 1.000,60
4) click off the fee amount box

witness that prior to the patch the net_amount is generated incorrectly (by js) but afterwards it i calculated correctly as 4.000,00

**Comment by colemanw** [comment]
Probably me. I can't spell that word for some reason.

**Pull request merged**